### PR TITLE
[Backport: v1.1.3] Fixed the log about Longhorn CSI plugin version

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#3079

### CODE REVIEW

File: csi/manager.go [hunk 1]
```diff
@@ -19,8 +19,8 @@ func GetCSIManager() *Manager {
 	return &Manager{}
 }
 
-func (m *Manager) Run(driverName, nodeID, endpoint, csiVersion, identityVersion, managerURL string) error {
-	logrus.Infof("CSI Driver: %v csiVersion: %v, manager URL %v", driverName, csiVersion, managerURL)
+func (m *Manager) Run(driverName, nodeID, endpoint, identityVersion, managerURL string) error {
+	logrus.Infof("CSI Driver: %v csiVersion: %v, manager URL %v", driverName, identityVersion, managerURL)
```
Comment (joshimoo) [inline csi/manager.go:23]:
```suggestion
	logrus.Infof("CSI Driver: %v version: %v, manager URL %v", driverName, identityVersion, managerURL)
```

Comment (joshimoo) [reply csi/manager.go:23]: Change it to just version, since it's not a the specific csi version instead it's the drivers version.

**Review by joshimoo — DISMISSED** [review]
LGTM, change the comment though to make it clear that it's the drivers version.

File: csi/manager.go (hunk 1 above)
Comment (PhanLe1010) [reply csi/manager.go]: Thanks @joshimoo Fixed it

**Review by innobead — APPROVED** [review]

**Pull request merged**